.changeset/serious-cougars-film.md

```diff
@@ -0,0 +1,7 @@
+---
+"@cube-creator/core-api": patch
+"@cube-creator/model": patch
+"@cube-creator/ui": patch
+---
+
+Redefine supported langauges and their order
```

Comment:
Typo and propose to add issue numbers where relevant so that they appear in the changelogs

```suggestion
Redefine supported languages and their order (#393)
```

Reply:
👍